Automattic/vip
# Allow imports for all sites with databases

## Description

Allow imports for all sites types with databases instead of hardcoding to a "WordPress" site type.

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip import sql`
